README.md

```diff
@@ -53,7 +53,42 @@ cd onyo
 pip install -e ".[tests, docs]"
 ```
 
-### Tests
+### Design
+
+Onyo follows a set of development and coding conventions throughout its code
+base. For linting and type checking, the Onyo project uses flake8 and Pyre.
+
+#### the complete code base
+- All classes, functions and properties have a doc-string describing their
+  functionality.
+- All functions have type hinting (including explicit return type for `None`).
+- The code is PEP8 conform.
+
+#### lib Tests (`onyo/lib/`)
+- they follow Onyos code conventions, see `"the complete code base"
```

Comment:
There's a stray backtick here.

What is meant by 'see "the complete code base"'? Do you mean that they should familiarize themselves with the surrounding code and follow its conventions?

Reply:
No, I meant to refer to the section five lines above `#### the complete code base`.
I will try to link to it, thanks for highlighting that this is ambigous when reading it ;)

Reply:
:facepalm: I /swear/ I looked for a section called "The complete code base".

That said, capitalizing it would make it more obvious that it's a title.